OrbitGl/LiveFunctionsController.cpp

```diff
@@ -207,6 +208,50 @@ void LiveFunctionsController::AddIterator(FunctionInfo* function) {
   Move();
 }
 
+void LiveFunctionsController::AddFrameTrack(const FunctionInfo& function) {
+  auto function_address = FunctionUtils::GetAbsoluteAddress(function);
+  std::vector<std::shared_ptr<TimerChain>> chains =
+      GCurrentTimeGraph->GetAllThreadTrackTimerChains();
+
+  std::vector<uint64_t> all_start_times;
+
+  for (auto& chain : chains) {
+    if (!chain) continue;
+    for (TimerChainIterator it = chain->begin(); it != chain->end(); ++it) {
+      TimerBlock& block = *it;
+      for (uint64_t i = 0; i < block.size(); i++) {
+        TextBox& box = block[i];
+        if (box.GetTimerInfo().function_address() == function_address) {
+          all_start_times.push_back(box.GetTimerInfo().start());
+        }
+      }
+    }
+  }
+  std::sort(all_start_times.begin(), all_start_times.end());
+
+  for (size_t k = 0; k < all_start_times.size() - 1; ++k) {
+    TimerInfo frame_timer;
+
+    // TID is meaningless for this timer (start and end can be on two different threads).
+    constexpr const int32_t kUnusedThreadId = -1;
```

Comment:
This might interfere with kAllThreadsFakeTid which is also -1.

Reply:
Yes, I thought about that.

My thinking was that:
1) There's not much I can do about it. I could use -3 here (as -2 is also already used), but there's no guarantee someone will not introduce -3 in some other context. 
2) It doesn't matter too much, as the timers are distinguished first based on type. We have other places where tids are used in a way that if timers would be added to the wrong track, they would mess up stuff. For example, the GPU job timers all have the tid of the submitting track. But since they are used only to populate the GPU track, this is fine. 

Let me know if you still have concerns and we can discuss if there's a better approach here.

Reply:
Yeah it's true that the dispatching of timers will occur based on the timer type, so this should not be a problem. I think it's fine.